apps/nextjs-website/src/components/atoms/InfoCardEditButton/InfoCardEditButton.tsx

```diff
@@ -0,0 +1,21 @@
+'use client';
+import { Button } from '@mui/material';
+import { useTranslations } from 'next-intl';
+import EditIcon from '@mui/icons-material/Edit';
+
+const InfoCardEditButton = ({ onClick }: { onClick?: () => null }) => {
+  const t = useTranslations('shared');
+
+  return (
+    <Button
+      onClick={onClick}
+      sx={{ paddingLeft: 0, paddingRight: 0 }}
```

Comment:
When the user clicks on the button, the lack of padding is a problem:
![image](https://github.com/pagopa/developer-portal/assets/39835990/c69a3349-6b89-438e-975a-085d3d9a8ee5).
I think you should use the ButtonNaked component